### ISSUE TITLE
Missing a step?

### ISSUE THREAD

**Issue by sirkitree** [Lullabot/react_form#1]
Are you planning on adding a build step to this? Currently the link does not work, and the source references a build directory that doesn't currently exist. Are you planning to add grunt or something to build/package this?

**Comment by juampynr** [comment]
It worked before when I had just an index.html file that used JSXProcessor. Now it is a Node.js app that needs the following to work:

```
npm install
node server.js
```

I am considering reverting these changes or move it to a server where I can start the node.js application (lulladev?).

**Comment by sirkitree** [comment]
I'd say that since it's an example for how to work with forms, and not how to compile jsx server side, I'd personally leave it as a client-side app entirely.

However, selfishly I'd love to see how this works server side as I'm at the point now where i have to convert my own app to work server side, which I've never done yet.

**Comment by juampynr** [comment]
@justafish suggested using Heroku or something similar. The benefit of this approach is that the form submission works. I remember that when we did the [AngularJS form article](https://www.lullabot.com/blog/article/processing-forms-angularjs), a lot of people wrote comments asking how to do the server side of things, which I have implemented here.

I will have a look at Heroku, it would be nice to have a Lullabot account there.
